rpm/fedora-27/docker-ce.spec

```diff
@@ -1,221 +0,0 @@
-Name: docker-ce
-Version: %{_version}
-Release: %{_release}%{?dist}
-Epoch: 3
-Summary: The open-source application container engine
-Group: Tools/Docker
-License: ASL 2.0
-Source0: engine.tgz
-Source1: cli.tgz
-URL: https://www.docker.com
-Vendor: Docker
-Packager: Docker <support@docker.com>
-
-# DWZ problem with multiple golang binary, see bug
-# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
-%global _dwz_low_mem_die_limit 0
-%global is_systemd 1
-%global with_selinux 1
-%global _missing_build_ids_terminate_build 0
```

Comment:
This line was in the Fedora specs, but not in the centos specs, and not in the current spec

Reply:
These don't actually matter all too much anymore since we don't build the daemon as part of the package building process, it's built as part of the engine image instead.